backend/tests/searchOffendersController.test.js

```diff
@@ -181,11 +181,11 @@ describe('Search offenders controller', () => {
                 value: '1:G0276VC:true',
```

Comment:
Really there ought to be a test to cover the ordering.

Reply:
Agreed. I've pulled those tests out into keyworkerShared.test.js